pom.xml

```diff
@@ -23,7 +23,8 @@
         <guava.version>27.0.1-jre</guava.version>
         <hamcrest.version>1.3</hamcrest.version>
         <jackson.version>2.6.5</jackson.version>
-        <junit.version>4.12</junit.version>
+        <!--        <junit.version>4.12</junit.version>-->
```

Comment:
Could the commented lines in the pom be removed?

Reply:
Thanks for spotting that - done